config/prod/s3-synapse-sync-bucket-a.yaml

```diff
@@ -12,7 +12,7 @@ hooks:
 dependencies:
   - "prod/s3-synapse-sync.yaml"
 parameters:
-  BucketName: "s3-synapse-sync-bucket-a"  # must match bucket name in data/s3-synapse-sync-bucket-vars.yaml
+  BucketName: "htan-synapse-sync-bucket-a"  # must match bucket name in data/s3-synapse-sync-bucket-vars.yaml
```

Comment:
as you probably know.  AWS does not allow changing a bucket name once it's been provisioned.  You probably want to create a new bucket and then delete  `htan-synapse-sync-bucket-a` bucket if you do not need it anymore.

Reply:
Right, we'll create a new bucket